src/Client.php

```diff
@@ -372,21 +384,25 @@ public function deletePlace($place)
     private function getUrl($url, array $query = null)
     {
         if ($query) {
+          foreach ($query as $key => $value) {
+            if (is_array($value)) {
+              $query[$key] = implode(',', $value);
+            }
+            if (is_bool($value)) {
+              $query[$key] = ($value) ? 'true' : 'false';
+            }
+          }
```

Comment:
Indentation seems to be off:

```suggestion
            foreach ($query as $key => $value) {
              if (is_array($value)) {
                $query[$key] = implode(',', $value);
              }
              if (is_bool($value)) {
                $query[$key] = ($value) ? 'true' : 'false';
              }
            }
```